src/Resources/config/resources.yml

```diff
@@ -0,0 +1,23 @@
+sylius_resource:
+  resources:
+    mango_order_comments_plugin.orderMessage:
+      classes:
+        model: MangoSylius\OrderCommentsPlugin\Entity\OrderMessage
+        form: MangoSylius\OrderCommentsPlugin\Form\Type\OrderMessageType
+
+swiftmailer:
+  transport: '%env(SYLIUS_MAILER_TRANSPORT)%'
+  host: '%env(SYLIUS_MAILER_HOST)%'
+  username: '%env(SYLIUS_MAILER_USER)%'
+  password: '%env(SYLIUS_MAILER_PASSWORD)%'
+
+
+sylius_mailer:
+  sender:
```

Comment:
remove sender, it does not belong to the plugin

Reply:
DONE